docs/Error-Handling.md

```diff
@@ -0,0 +1,37 @@
+<h1 align="center">Fastify</h1>
+
+## Error classification in Promises & Async/Await
+
+Fastify classified errors in two different groups.
+
+- `System` any instance of type `EvalError`, `RangeError`, `ReferenceError`, `SyntaxError`, `TypeError`, `URIError` or `AssertionError`
```

Comment:
`- System: ...`
`- Business: ...`

Note the lack of backticks and the addition of a colon.